<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Opp1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Opp1</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="player">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Opp2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Opp2</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TM">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TM</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card12">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="card13">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>25</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="play1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play12">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="play13">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>play</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>471</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bidBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="bidButton">
       <property name="text">
        <string>Bid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="turnButton">
       <property name="text">
        <string>Next Turn</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
